1 Foundations/this.js

```diff
@@ -1,37 +1,160 @@
-// Definition of 'this'
-// 'this' is an object that the function is a property of.
```

Comment:
This definition should have been let be. This is something very short and explains a lot.